jboss-eap-standalone-rhel8/README.md

```diff
@@ -82,20 +81,20 @@ Build your environment with EAP 7.2 on top of RHEL 8.0 on Azure in a few simple
 
    - **EAP Password** - User account password for EAP Console.
     
-   - **RHSM Username** - User name for RedHat account.
+   - **RHSM Username** - User name for Red Hat account.
 
    - **RHSM Password** - User account password for RedHat account.
     
    - **SSH Key Data** - Generate an SSH key using Puttygen and provide the data here.
 
-   - Leave the rest of the Parameter Value as it is and proceed.
+   - Leave the rest of the Parameter Values (artifacts and Location) as it is and proceed to purchase.
```

Comment:
Modified this line as follows. Please suggest

- Leave the rest of the parameter values (artifacts and Location) as is and accept the terms and conditions before clicking on Purchase